### ISSUE TITLE
[Bug]: clicking any most visited site opens a new tab

### ISSUE THREAD

**Issue by bjoli** [mozilla-mobile/fenix#23452]
### Steps to reproduce

1. Have a tab at any website, like firefox.com
2. Click the url bar
3. Click one of the most visited sites

### Expected behaviour

The most visited website should open in the active tab, like it does when typing in an adress manually or searching. Like it did since forever, in every browser.

### Actual behaviour

The most visited website you clicked will open in a new tab, always behaving as if firefox was newly started with no tabs open.

### Device name

google pixel 5a

### Android version

android 12

### Firefox release type

Firefox

### Firefox version

96.2.0 (Build #2015858259)

### Device logs

I have managed to reproduce it on 2 devices (pixel 5a and Samsung a52 5g). I suspect it might be universal. If not I will of course provide a log.

### Additional information

This affects 2 devices. A pixel 5a (android 12) and a samsung a52 (android unknown) and I believe it was introduced in the latest update.

**Comment by marcel263** [comment]
this is by design

**Comment by bjoli** [comment]
Is it configurable? If I want to go from firefox.com to one of my most visited websites, am I doomed to either having to write the address in or going back to close the original tab each time? It would then seem more coherent to make writing an address in also open in a new tab.

I mean, I can't be the only one with 90% of my web browsing being on about 7 different pages. This change now means that the tabs I have saved for later now are littered with multiple copies of the same 7 websites.
